pkgs/development/tools/misc/ccache/default.nix

```
@@ -98,12 +98,12 @@ stdenv.mkDerivation (finalAttrs: {
           fi
         }
 
-        wrap cc
-        wrap c++
-        wrap gcc
-        wrap g++
-        wrap clang
-        wrap clang++
+        wrap ${binutils.targetPrefix}cc
+        wrap ${binutils.targetPrefix}c++
+        wrap ${binutils.targetPrefix}gcc
+        wrap ${binutils.targetPrefix}g++
+        wrap ${binutils.targetPrefix}clang
+        wrap ${binutils.targetPrefix}clang++
```

```suggestion
        wrap ${stdenv.cc.targetPrefix}cc
        wrap ${stdenv.cc.targetPrefix}c++
        wrap ${stdenv.cc.targetPrefix}gcc
        wrap ${stdenv.cc.targetPrefix}g++
        wrap ${stdenv.cc.targetPrefix}clang
        wrap ${stdenv.cc.targetPrefix}clang++
```

---

well this is not really correct either

---

```suggestion
        wrap ${lib.optionalString unwrappedCC.targetConfig "${unwrappedCC.targetConfig}-"}cc
        wrap ${lib.optionalString unwrappedCC.targetConfig "${unwrappedCC.targetConfig}-"}c++
        wrap ${lib.optionalString unwrappedCC.targetConfig "${unwrappedCC.targetConfig}-"}gcc
        wrap ${lib.optionalString unwrappedCC.targetConfig "${unwrappedCC.targetConfig}-"}g++
        wrap ${lib.optionalString unwrappedCC.targetConfig "${unwrappedCC.targetConfig}-"}clang
        wrap ${lib.optionalString unwrappedCC.targetConfig "${unwrappedCC.targetConfig}-"}clang++
```
Should be something like this
Put it in a let